examples/stm32/xively_demo_mt/Src/system_stm32f4xx.c

```diff
@@ -1,21 +1,21 @@
 /**
   ******************************************************************************
-  * @file    system_stm32f4xx.c

```

Comment:
Yeah, It's not my best idea... so question here: 

Can we just make it explicit that this example require version x.xx.x of SDK and that's it ?